src/features/review/components/dashboard/entomological-summary-section.tsx

```diff
@@ -28,88 +28,90 @@ export function EntomologicalSummarySection({
 
   const hasData =
     speciesDistribution.length > 0 ||
-    sexRatio.total > 0 ||
-    abdomenStatus.total > 0;
+    sexDistribution.total > 0 ||
+    abdomenStatusDistribution.total > 0;
 
-  // Figma-aligned palette: shades of green only
-  const sexColors = ['#166534', '#22C55E']; // dark green, medium green
-  const abdomenColors = ['#166534', '#22C55E', '#86EFAC']; // dark, medium, light green
+  const sexColors = [
+    'var(--color-chart-green-dark)',
+    'var(--color-chart-green-medium)',
+  ];
+  const abdomenColors = [
+    'var(--color-chart-green-dark)',
+    'var(--color-chart-green-medium)',
+    'var(--color-chart-green-light)',
+  ];
```

Comment:
You dont need to define the colors like 'var(--color-chart-green-dark)' if you have defined it in globals.css

All you have to do is use the tailwindcss class name in the className prop of any component